src/ipvs.c

```diff
@@ -302,8 +322,16 @@ static void cipvs_submit_service (struct ip_vs_service_entry *se)
 	cipvs_submit_if (pi, "if_packets", NULL, stats.inpkts, stats.outpkts);
 	cipvs_submit_if (pi, "if_octets", NULL, stats.inbytes, stats.outbytes);
 
+	size_t live_dests = 0;
 	for (size_t i = 0; i < dests->num_dests; ++i)
+	{
+	if (dests->entrytable[i].weight != 0)
+		++live_dests;
 		cipvs_submit_dest (pi, &dests->entrytable[i]);
+	}
+
+	cipvs_submit_dest_count(pi, "total", dests->num_dests);
```

Comment:
One of our best practices is to report metrics that can reasonably be summed up, in this case report "active" and "inactive" (which can usually quite easily be summed up to a "total") rather than report "total" and "active" since subtracting is usually much harder in graphing front-ends.